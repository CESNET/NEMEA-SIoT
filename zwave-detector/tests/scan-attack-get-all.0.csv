time,time TIMESTAMP,uint8 CHANNEL,bytes FRAME
2020-01-19T22:57:00,2020-01-19T22:57:00.000,2,00
2020-01-19T22:57:02,2020-01-19T22:57:02.000,2,e14534730101010c02720465
2020-01-19T22:57:02,2020-01-19T22:57:02.000,2,e14534730101010c02861184
2020-01-19T22:57:02,2020-01-19T22:57:02.000,2,e14534730101010c02010210
2020-01-19T22:57:02,2020-01-19T22:57:02.000,2,e14534730101010c02200231
2020-01-19T22:57:02,2020-01-19T22:57:02.000,2,e14534730101010d0270050166
