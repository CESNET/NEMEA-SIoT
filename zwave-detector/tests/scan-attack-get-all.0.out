2,2020-01-19T22:57:02.000,1,"Scanned: Device Manufacturer and Device Type, Device Software Version, Supported Command Classes, Basic Operational Status, Configuration Settings,"
